styleguide/src/theme.ts

```diff
@@ -1,8 +1,47 @@
 import { common, green, grey } from '@material-ui/core/colors'
 import { darken } from '@material-ui/core/styles/colorManipulator'
+import { ThemeOptions } from '@material-ui/core/styles/createMuiTheme'
 import spacing from '@material-ui/core/styles/spacing'
 
-export const theme = {
+declare module '@material-ui/core/styles/createPalette' {
```

Comment:
These were resolved via https://material-ui.com/guides/typescript/#customization-of-theme. But could definitely use some eyes on this since I'm not familiar with CSS styling. @NavyAdmiral  @rupurt

Reply:
Awesome! Been annoying not having types for these